src/coreComponents/physicsSolvers/fluidFlow/wells/CompositionalMultiphaseWell.hpp

```diff
@@ -140,10 +140,10 @@ class CompositionalMultiphaseWell : public WellSolverBase
 
   /**
    * @brief Recompute the volumetric rates that are used in the well constraints
+   * @param elemManager the well region manager containing the well
    * @param subRegion the well subregion containing all the primary and dependent fields
-   * @param targetIndex the targetIndex of the subRegion
    */
-  void updateVolRatesForConstraint( WellElementSubRegion & subRegion );
+  void updateVolRatesForConstraint( ElementRegionManager const & elemManager, WellElementSubRegion & subRegion );
```

Comment:
(same for `SinglePhaseWell`)
```suggestion
  void updateVolRatesForConstraint( ElementRegionManager const & elemManager,
                                    WellElementSubRegion & subRegion );
```